LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
[The PNext project is under the BSD-Clause-2 license](https://github.com/pnext/org#license). That being said, this is a code-license and not a Data license. Which should be clarified somewhere.